src/Microsoft.AspNetCore.Razor/CodeGenerators/Visitors/CSharpCodeVisitor.cs

```diff
@@ -399,29 +399,38 @@ protected override void Visit(SectionChunk chunk)
 
         public void RenderDesignTimeExpressionBlockChunk(ExpressionBlockChunk chunk)
         {
-            var firstChild = (ExpressionChunk)chunk.Children.FirstOrDefault();
-
+            var firstChild = chunk.Children.FirstOrDefault();
             if (firstChild != null)
```

Comment:
I know you didn't write this but since we're touching bits around this; to avoid lots of indentation lets do:

```C#
if (firstChild == null)
{
    return;
}
```